README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
I don't think it's that straight forward. Take `Flip f a b`. How these variables are used depends on `f`. If you have

```haskell
newtype Reader a b = Reader (a -> b)
newtype Op a b = Op (b -> a)
```

Then, `Flip Reader a b` has the `a` and `b` in negative and positive position, respectively.
But, `Flip Op a b` has the `a` and `b` in positive and negative position, respectively.
All while `Flip`s kind is still `* -> * -> *`.

There may be some relationship though.